setup_ubuntu.sh

```diff
@@ -0,0 +1 @@
+apt-get install python-pyaudio python-wxGlade
```

Comment:
Yeah, haven't touched that part. That file is from MaskedRetriever. Maybe he wanted to try around a bit with wxGlade. I would say this is more something that would fit into the Readme.